Tests/LinearSolvers/CurlCurl/initProb_K.H

```diff
@@ -0,0 +1,107 @@
+#ifndef INIT_PROB_K_H_
+#define INIT_PROB_K_H_
+
+#include <AMReX_FArrayBox.H>
+
+AMREX_GPU_DEVICE AMREX_FORCE_INLINE
+void actual_init_prob (int i, int j, int k,
+                       amrex::GpuArray<amrex::Array4<amrex::Real>,3> const& rhs,
+                       amrex::GpuArray<amrex::Array4<amrex::Real>,3> const& sol,
+                       amrex::GpuArray<amrex::Real,AMREX_SPACEDIM> const& problo,
+                       amrex::GpuArray<amrex::Real,AMREX_SPACEDIM> const& dx,
+                       amrex::Real alpha, amrex::Real beta)
+{
+    using namespace amrex;
```

Comment:
To simplify a few of the `Real(...)` with `..._rt`:
```suggestion
    using namespace amrex;
    using namespace amrex::literals;
```

Reply:
I found `Real` to be easier for manipulation with regex.

Reply:
So I usually prefer Real to _rt.